circuit-retained/src/androidMain/kotlin/com/slack/circuit/retained/AndroidContinuity.kt

```diff
@@ -65,9 +65,10 @@ internal class Continuity : ViewModel(), RetainedStateRegistry {
  */
 @Composable
 public fun continuityRetainedStateRegistry(
-  factory: ViewModelProvider.Factory = Continuity.Factory
+  factory: ViewModelProvider.Factory = Continuity.Factory,
+  key: String = Continuity.KEY,
```

Comment:
nit: let's match the same order as they're used in the other vm factory APIs

```suggestion
  key: String = Continuity.KEY,
  factory: ViewModelProvider.Factory = Continuity.Factory,
```

Reply:
will that not be a breaking change for existing usages?

```kotlin
continuityRetainedStateRegistry(SomeViewModelProvider.Factory)
```

Reply:
it would be, but we are not a stable API currently :)

Reply:
lfg then